web_server/social_distribution/utils/basic_auth.py

```diff
@@ -54,11 +54,12 @@ def handle_incoming_request_or_request(view, request, test_func, realm="", *args
 
                 # Check if the credentials are valid for the host requesting them
                 if Node.objects.all().filter(foreign_server_username=uname).exists():
-                    entry = Node.objects.get(foreign_server_username=uname)
+                    entry = Node.objects.all().get(foreign_server_username=uname)
+
                     if check_password(passwd, entry.foreign_server_password):
                         return view(request, *args, **kwargs)
                     else:
-                        deny_response += " The provided password for your server authentication was invalid"
+                        deny_response += " The provided password for yours server authentication was invalid"
```

Comment:
Why this change? This takes it from correct grammar to incorrect grammar. Please undo this change.

Reply:
i don't remember I changed that.  lol. anyway i will remove "s"